spirv_msl.cpp

```diff
@@ -125,10 +119,30 @@ string CompilerMSL::compile(MSLConfiguration &msl_cfg, vector<MSLVertexAttr> *p_
 	return buffer->str();
 }
 
-string CompilerMSL::compile()
+string CompilerMSL::compile(vector<MSLVertexAttr> *p_vtx_attrs, std::vector<MSLResourceBinding> *p_res_bindings)
```

Comment:
Removed unnecessary use of std:: in cpp file.

Incoming vectors cannot be marked as `const` because the values in the vectors may be modified during compilation to have their `used_by_shader` flag set. This is documented in constructor API.